www/blog-roundup/2023-03-20-thismonth/index.md

```diff
@@ -0,0 +1,120 @@
+* **March 16** | `youtube.com` - [**Connect to a Database directly from Static Web Apps**](https://youtu.be/vGOnh0UrADg) by _Thomas Gauvin_. Discover how to connect your Azure SQL Database to your Static Web Apps, and access it through a secure data endpoint generated for you. This video shows how to setup the connection, and use the REST API to perform Create, Read, Update and Delete (CRUD) operation on your data.
+* **March 16** | `youtube.com` -  [**Query a Database using GraphQL from your Static Web Apps**](https://youtu.be/NF0EC68rdQk) by _Craig Shoemaker_. Learn how to query your Database directly through your Static Web Apps using GraphQL, without having to write any backend code with Azure Static Web Apps database connections feature. This video shows how to setup and use the database connection feature to use GraphQL queries in your apps.
+
+---
+
+## Community Buzz
+
+:::info AUTHORED BY DEVS LIKE YOU!
+_This section highlights content from our amazing developer community - submitted directly, or discovered via the Azure Static Web Apps tag in sites like Tech Community and dev.to._
+:::
+
+* **March 13** | `dev.to` - [**Astro and Azure Static Web Apps**](https://dev.to/florianrappl/astro-and-azure-static-web-apps-efl) by _Florian Rappl_ In this article what Astro is, and why it is ideal for an information driven static website. Then dive into Azure SWA as a hosting candidate, and what you might want to consider when going there, and how to deploy your Astro site.
+* **March 10** | `geeksforgeeks.org` -  [**How to use Azure Static Web Apps to Deploy an Angular App**](https://www.geeksforgeeks.org/how-to-use-azure-static-web-apps-to-deploy-an-angular-app/) by _Manikandan Shanmugam_ Follow this step by step guide on how you can create and deploy your own Angular app to Azure Static Web Apps with having just a GitHub and an Azure account.
+* **March 20** | `blog.johnnyreilly.com` -  [**Playwright, GitHub Actions and Azure Static Web Apps staging environments**](https://blog.johnnyreilly.com/playwright-github-actions-and-azure-static-web-apps-staging-environments) by _John Reilly_ I was trying to setup a deployment for an SPA application to an Azure Static Web app using TeamCity, but most of the tutorials I found was doing the deployment through CI/CD of Github's Actions .. But what if .. we need to do it on demand from TeamCity, or any other CI/CD tool? You can do this using [the Azure Static Web Apps CLI](https://github.com/Azure/static-web-apps-cli)! Learn how.
+
+---
+
+## Upcoming Events
```

Comment:
Do we not wanna call out Student Summit? We can give it more real estate possibly next month, but a mention might be good and timely right?

Reply:
Thanks for thinking of Summit! I didn't include it initially as some regional events have already completed, but they have the recording already linked up. Getting this added in now 😊